tests/helper/skyMapTestCase.py

```diff
@@ -30,6 +30,30 @@
 from lsst.skymap import skyMapRegistry
 
 
+def checkDm14809(testcase, skymap):
+    """Test that DM-14809 has been fixed
+
+    The observed behaviour was:
+
+        skyMap.findTract(skyMap[9712].getCtrCoord()).getId() != 9712
+
+    and
+
+        skyMap[1].getCtrCoord() == skyMap[11].getCtrCoord()
+
+    In order to be thorough, we generalise these over the entire skymap.
+    """
+    # Check that the tract ID is that used to index the skymap
+    expect = [tract.getId() for tract in skymap]
+    got = [skymap.findTract(tract.getCtrCoord()).getId() for tract in skymap]
+    testcase.assertListEqual(got, expect)
+
+    # Check that the tract found for central coordinate of a tract is that tract
```

Comment:
I think this comment applies to the test above, not the one below, which seems to assert that no two tracts have the same (numerically identical) centers, with no "find" involved.

I'm not sure this 2nd test is actually required, but I think it would be a lot more useful if it tested that no two tracts had nearly identical centers. One efficient way to do that is to round the RA and Dec values to some reasonable small value before putting the tuple in the set.